### ISSUE TITLE
[Modals]  return outputs in addViewSubmissionHandler

### ISSUE THREAD

**Issue by RachadAbiChahine** [slackapi/deno-slack-sdk#262]
**Question**

I am trying to use multi step workflow to create an github issue,
In my case i want to use modal to make a primary choice if the client want to create an issue or just post a question
<img width="285" alt="image" src="https://github.com/slackapi/deno-slack-sdk/assets/13370993/6606a7c4-4a12-41ff-a614-eaefd019f8f8">

Then another modal will be opened with the correct inputs using `addBlockSuggestionHandler` then user will submit the form

then  with `addViewSubmissionHandler` I want to be able to collect the inputs and send it to the next workflow step 

but I can't find a way to do this 

`const product = view.state.values?.[BLOCK_PRODUCT_SELECT_ID]?.[BLOCK_PRODUCT_SELECT_ACTION_ID].selected_option?.value; 
...
return {
                output:{product:product},
                response_action: "clear",
                view: {}
            };`
            
But when I add outpt I get an error 
<img width="498" alt="image" src="https://github.com/slackapi/deno-slack-sdk/assets/13370993/184bd110-4842-45c0-a269-21f9906fb501">

what is the right way to pass the result of the submited value to the next workflow step ?

**Comment by filmaj** [comment]
@RachadAbiChahine it looks to me like you are trying to return function outputs in your code (based on `return { output: {} ... }`) from within an interactivity handler such as a View Submission Handler.

I do not think that is possible to do directly using `return` from an interactivity handler. Any direct `return`s from an interactivity handler only affect the specific interactivity element/event using the `response_action` object property. More details on this are available in our [Interactivity docs](https://api.slack.com/automation/interactive-modals).

I have an example interactivity app on GitHub, and from the interactivity handlers in this app (see [this code](https://github.com/filmaj/interactive-approval/blob/main/functions/approval/mod.ts#L210-L213)), to set the `outputs` on the housing function from one of its interactivity handlers, you must use the asynchronous API client and call `functions.completeSuccess` (if the function execution completed successfully) or `functions.completeError` (if the function execution failed).

I realize that the [Interactivity docs](https://api.slack.com/automation/interactive-modals) imply that you may `return` function outputs directly from the interactivity handlers, but that is a mistake. I will follow up with the documentation team to ensure this is removed.

**Comment by filmaj** [comment]
Yes that is most definitely possible. I am confused why you say it forces you to organize your code in a specific, less-readable way. Can you post an example of what you mean to help me understand?
